Sources/Rules.swift

```diff
@@ -5171,15 +5171,21 @@ public struct _FormatRules {
         //    and it's more difficult to track the usage of other declaration
         //    types like `init`, `subscript`, `operator`, etc.
         let allowlist = ["let", "var", "func", "typealias"]
+        let overrideList: [DeclarationType] = [.overriddenProperty, .overriddenMethod]
 
         // Collect all of the `private` or `fileprivate` declarations in the file
         var privateDeclarations: [Declaration] = []
         formatter.forEachRecursiveDeclaration { declaration in
             guard allowlist.contains(declaration.keyword),
                   let name = declaration.name,
-                  !(formatter.options.preservedPrivateDeclarations.contains(name))
+                  !(formatter.options.preservedPrivateDeclarations.contains(name)),
```

Comment:
These parens are unnecessary

```suggestion
                  !formatter.options.preservedPrivateDeclarations.contains(name),
```